module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -1312,7 +1312,10 @@ public function placeHold($holdDetails)
         );
 
         if ($result['code'] >= 300) {
-            return $this->holdError($result['data']['error'] ?? 'hold_error_fail');
+            if (!empty($result['data']['error'])) {
+                return $this->holdError($result['data']['error']);
+            }
+            return $this->holdError('hold_error_fail', false);
```

Comment:
Code simplification:

```suggestion
            return empty($result['data']['error'])
                ? $this->holdError('hold_error_fail', false)
                : $this->holdError($result['data']['error']);
```